### PR TITLE
Support Python 3: initial ops

### PR DESCRIPTION
**Pull request by adam-thomas**
Expanding the test coverage (see https://github.com/incuna/django-admin-sso/issues/4) and adding Python 3 support to the untested stretches of the code go hand in hand, so I can't really add Python 3 support all in one go (unless it miraculously works) with one issue.  This issue is about P3 support before I expand tests - i.e. when the initial six tests in test_auth all pass on Python 2 and 3, this can be reviewed and merged.

### CODE REVIEW

**Comment by adam-thomas** [comment]
Don't merge yet - I'm using the issue so I can get Travis to run tests for me :)

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,19 @@ language: python
...
-  - DJANGO_VERSION="django>=1.6,<1.7"
+  matrix:
+    - DJANGO_VERSION="django>=1.4,<1.5"
+    - DJANGO_VERSION="django>=1.5,<1.6"
+    - DJANGO_VERSION="django>=1.6,<1.7"
+matrix:
+  exclude:
+     - env: DJANGO_VERSION<1.5
+       python: 3.3
+     - env: DJANGO_VERSION<1.5
```
Comment (meshy) [inline .travis.yml]: I think this will need to be `DJANGO_VERSION="django>=1.4,<1.5"`

Comment (meshy) [inline .travis.yml]: Also, I think your multi-line yaml syntax may be wrong

Comment (meshy) [inline .travis.yml]: (maybe)

Comment (adam-thomas) [inline .travis.yml]: I think the syntax is ok (I copied it out of another file), but needing it to be `DJANGO_VERSION="django>=1.4,<1.5"` appears to be the case.

Comment (meshy) [inline .travis.yml]: Cool :+1:

**Comment by adam-thomas** [comment]
screenshot from 2014-05-19 10 59 45

Excellent.  Let's get started.

**Comment by meshy** [comment]
https://i.chzbgr.com/maxW500/4783401216/hED3700B5/

File: admin_sso/store.py [hunk 2]
```diff
@@ -71,11 +71,21 @@ def getAssociation(self, server_url, handle=None):
         associations = []
         expired = []
         for assoc in assocs:
+            print(assoc.secret)
+            print(type(assoc.secret))
```
Comment (meshy) [inline admin_sso/store.py]: Couple of stray `print`s here `;)`

Comment (adam-thomas) [inline admin_sso/store.py]: ...oops :P

**Comment by adam-thomas** [comment]
Made a pull request to the real deal repository (after a rebase to tidy up the history) at https://github.com/frog32/django-admin-sso/pull/12.

**Comment by adam-thomas** [comment]
Closing this since the replacement fork at https://github.com/matthiask/django-admin-sso supports Python 3.

**Pull request closed without merging**